src/doctor.ts

```diff
@@ -0,0 +1,80 @@
+import { getProtocolInterface } from "./deps.ts";
+import { isNewSemverRelease } from "./utilities.ts";
+
+type RuntimeVersion = {
+  name: string;
+  current: string;
+} & RuntimeDetails;
+
+type RuntimeDetails = {
+  message?: string;
+  error?: {
+    message: string;
+  };
+};
+
+const getHostedDenoRuntimeVersion = async (): Promise<RuntimeDetails> => {
+  try {
+    const metadataURL = "https://api.slack.com/slackcli/metadata.json";
+    const response = await fetch(metadataURL);
+    if (!response.ok || response.status !== 200) {
```

Comment:
The `response.status !== 200` protects against successful but unexpected responses, but maybe we want to strengthen this?

The actual fetch follows redirects which I think is good for future compatibility, but that means this can be redirected (302) to a successful page (200) which is not the right page. For instance, a login page. Then the result is this:

![login](https://github.com/slackapi/deno-slack-hooks/assets/18134219/4771db5e-fcc3-4031-a8d0-4b62da343ee8)